### ISSUE TITLE
AMQPLAIN not supported 

### ISSUE THREAD

**Issue by s-maj** [cloudamqp/amqproxy#12]
I am getting from Celery(4.2.1)/Kombu(4.2.1)/PyAMQP(2.3.2):
`ConnectionError: Couldn't find appropriate auth mechanism (can offer: AMQPLAIN; available: PLAIN)` but it looks like AMQPLAIN should be supported https://github.com/cloudamqp/amqproxy/blob/be42d09ce48d94211f8c202aed3b90fc5cf1a007/src/amqproxy/client.cr#L67
From AMQProxy side:
```
Unhandled exception in spawn: AMQ::Protocol::Frame::Body-end was 10, expected 206 (AMQ::Protocol::Error::InvalidFrameEnd)
  from src/amqproxy/client.cr:42:15 in '->'
  from /usr/share/crystal/src/fiber.cr:255:3 in '???'
  from ???
```

**Comment by carlhoerberg** [comment]
Sorry about that, fixed here https://github.com/cloudamqp/amqproxy/commit/0e6d6432f52be83cc4da737d9872c4e1c15e76a0

v0.3.1 released: https://github.com/cloudamqp/amqproxy/releases/tag/v0.3.1

**Comment by carlhoerberg** [comment]
Actually that was another problem, but this issue is fixed in 0.3.2

**Comment by s-maj** [comment]
Ok, looks good. 
I've opened two more issues:
https://github.com/cloudamqp/amqproxy/issues/14
https://github.com/cloudamqp/amqproxy/issues/13
